specification/compute/resource-manager/Microsoft.Compute/preview/2020-09-30/sharedGallery.json

```diff
@@ -532,7 +532,7 @@
           ],
           "x-ms-enum": {
             "name": "HyperVGeneration",
-            "modelAsString": false
+            "modelAsString": true
```

Comment:
this will be SDK breaking change

Reply:
Thanks for review! I need to make this change for a successful SDK generation. This is in preview, and no one is using it.

Reply:
thanks for confirmation,  so No sdk released yet?

Reply:
Yes. Some tiny parts are preventing me from doing sdk generation, so I am making this pr. No sdk released yet.

Reply:
thanks for confirmation.